Sources/SQLKit/Query/SQLAlterTable.swift

```diff
@@ -31,7 +32,11 @@ public struct SQLAlterTable: SQLExpression {
                 $0.append(column)
             }
             for column in self.modifyColumns{
-                $0.append("MODIFY")
+                if dialect.name == "mysql" {
```

Comment:
Shouldn't we have a `SQLDialect.alterColumn` property, instead of just checking the dialect name?

Reply:
Yes definitely.

Reply:
I'm not 100% sure on the context here for this support. But is modify the only option you want to support here? You can't change a column name with this option I believe, but `CHANGE` will allow this. Although it has tradeoffs too with renaming stuff twice etc. Just a question :slightly_smiling_face:

Reply:
Yeah I need to take a closer look at what all the dialects support. At a minimum, for the fluent feature this is targeting, I need to support modifying a column's data type. I think I'm gonna try to keep it as close to that minimum requirement as possible.

Reply:
That makes sense if it's just a type change 🙂